selfdrive/car/subaru/carcontroller.py

```diff
@@ -70,20 +130,42 @@ def update(self, CC, CS):
         self.es_distance_cnt = CS.es_distance_msg["COUNTER"]
 
     else:
-      if pcm_cancel_cmd and (self.frame - self.last_cancel_frame) > 0.2:
-        bus = 1 if self.CP.carFingerprint in GLOBAL_GEN2 else 0
-        can_sends.append(subarucan.create_es_distance(self.packer, CS.es_distance_msg, bus, pcm_cancel_cmd))
-        self.last_cancel_frame = self.frame
-
       if self.es_dashstatus_cnt != CS.es_dashstatus_msg["COUNTER"]:
-        can_sends.append(subarucan.create_es_dashstatus(self.packer, CS.es_dashstatus_msg))
+        can_sends.append(subarucan.create_es_dashstatus(self.packer, CS.es_dashstatus_msg, CC.enabled, hud_control.leadVisible))
```

Comment:
Should `hud_control.leadVisible` only be passed in if OP longitudinal is enabled?

I've got this PR applied in my personal branch, and you see the `Car_Follow` dash image flashing (to match openpilot's model seeing a lead vehicle) even if OP long is disabled and stock long is in use.

Ideally on stock long, it just passes through whatever is in `Car_Follow`

Reply:
thanks, I updated create_es_dashstatus so long related signals are only set when openpilot long is enabled